### DISCUSSION TITLE
[Suggestion] Squad mechanic can be way better and more intuitive

### DISCUSSION THREAD

**Discussion by mrvice** [KnuckleCracker/CW4-bug-tracker#372] [Ideas]
Please make sure you have read, understood and agreed to the [policy](https://github.com/KnuckleCracker/CW4-bug-tracker/blob/master/Suggestion%20policy.md) regarding submitting material to Knuckle Cracker

**Is your feature request related to a problem? Please describe.**
The current squad mechanic is horrible and very unintuitive

**Describe the solution you'd like**
Like every other RTS has done it.
Select Units u want in squad
press "shift+numberkey" example "shift+1" "shift+2" and so on
pressing 1 should selec your squad
pressing 1 two times in a row should move camera to selected squad.

**Describe alternatives you've considered**

**Additional context**

**Comment by jcheung9941** [comment]
but currently 1-8 or however high it goes is used to quick-select items for building. me personally, i rarely ever actually use an actual squad or formation moves, but i use those number keys often to build with... so where would those go?
i might agree with numpad doing what you're saying though

**Comment by mrvice** [comment]
Hmm... then maybe just implement the option with the modifier key "shift" and we will assign stuff to our liking.
but youre right with the nubmer keys... numpad tho is way to far away.
But as long as its costum bindable every player can work that out for themselves.
for exmaple using QWERT  or ASDF and so on would also be an option.
but again as long as its costumizeable there shouldn´t be an issue.

**Comment by Karsten75** [comment]
My objection to the numbered squad keys is that you have to remember which units are in which squad. 

The way this game has it now, you know approximately where each squad is. Click on a unit, you see it's in a squad, you click to select (this maybe could be a hotkey) and now you have the entire squad. 

I usually allocate bombers and AC Bombers in squads and they all are in approximately the same general location, so it's much easier this way, at least for me.

**Reply by mrvice** [reply]
the fact that i have to know wich unit is in wich squad and then first find it and click on it and then select squad is just to many actions for such a simple task.
----
could be so much easier when you hit the hotkey for squad 1 the units get selectet and therfore highlighted.

for example if i currently have bombers in a squad iam force do click on the launchpad and then i need to move the mouse accross half of my screen only to select the whole squad.

hotkeys for multiple squad would be so much more enjoyable.

**Comment by Karsten75** [comment]
Based on a discussion we just had in the Discord, I think I now understand your problem better. 

You go back to the bomber base to select bombers. 

What you can do is click on the target marker for any bomber, that will also bring up  the "select squad" so no panning across  the map. And no need to remember which bombers you have assigned to which squad. 

Try that, let me know if that at least partially solve your problem.

**Reply by mrvice** [reply]
i mean yeah i could do that but it still would not solve the more convinient squad create squad choose stuff via hotkeys
There are instances where i have like over 15 target markers targeting different stuff sometimes interconnecting in a specific pattern.
 and then i probably still have to scroll back to look if i have selected the correct squad.

EDIT: but yeah keep in mind i only give a suggestion here that it find it is maybe a good idea to have a system like that wich is litterly present in every other strategy game.
But if you don´t think it would be easy to implement or benefit the game then it´s also ok ^^
but still thx for your advices :)

**Comment by Karsten75** [comment]
I'm not saying you are wrong. In CW3 there were numbered squad keys. Primarily, one of our concerns were that the numbered keys were getting overloaded. It also potentially limited the number of squad assignments. 

Thank you for your time and patience explaining how you use squads assignments.
